pig_the_dice.py

```diff
@@ -7,53 +7,72 @@
 player['com'] = 0
 winner = ''
 
-while(True):
+def roll():
+    n = rand.randrange(1,7)
+    print(f'your number is {n}')
+    return n
+
+def stop(score, i):  
+    player[i] += score
+    print(f'your turn is over. score {score} saved.')
+    
+def winner_check(i):
+    if player[i] >= 30:
+        winner = str(i)
+        return winner
+    else:
+        return 0
+    
+def calc_score(score, n):
+        score += n
+        print(f'total score in this turn: {score}')
+        return score    
 
+while(True):
+    
+    if winner in player:
+        print(player)
+        print(f'winnder is {winner}!')
+        break
+    
     for i in player.keys():
         score = 0
+        
+        if winner in player:
+            break
 
         while(True):
             print(f'{i} players turn.')
             if i == 'com':
-                for roll in range(rand.randrange(0,10)):
```

Comment:
could you change computers play 10times to 5times?

Reply:
sure!